modules/ROOT/partials/configuration/end_container_on_empty_block.adoc

```diff
@@ -1,20 +1,26 @@
 [[end_container_on_empty_block]]
 == `+end_container_on_empty_block+`
 
-This option allows you to to split the current container block element if the enter key is pressed inside an empty inner block element.
+This option specifies how the current container block element is split when the Enter key is pressed inside an empty inner block element. The available options are:
 
-*Type:* `+Boolean+`
+* always split the block;
+* split for specific block elements; or
+* do not split the block at all.
 
-*Default value:* `+false+`
+*Type:* `+Boolean+` or `+String+`
 
-*Possible values:* `+true+`, `+false+`
+*Default value:* `+'blockquote'+`
+
+*Possible values:* `+true+`, `+false+` or comma separated `+String+` of element tag names
```

Comment:
Oops sorry Mitch, just noticed I had a grammar error in the suggestion I gave previously 😞 
```suggestion
*Possible values:* `+true+`, `+false+` or a comma separated `+String+` of element tag names
```